### PR TITLE
Created Instant Logs page

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: products/logs/src/content/instant-logs.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+pcx-content-type: concept
+---
+
+# Instant Logs
+
+Instant Logs allows Cloudflare Enterprise customers to access a live stream of the traffic for their domain on the Cloudflare dashboard. Seeing data in real time allows you to investigate an attack, troubleshoot, debug or test out changes made to your network. Instant Logs is lightweight, simple to use and does not require any additional setup.
+
+## How to use Instant Logs
+
+To use Instant Logs:
```
Comment (marciocloudflare) [inline products/logs/src/content/instant-logs.md]: Consider deleting this sentence. It's redundant

File: products/logs/src/content/instant-logs.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Log in to the Cloudflare dashboard.
```
Comment (marciocloudflare) [inline products/logs/src/content/instant-logs.md]:
```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login).
```

File: products/logs/src/content/instant-logs.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Select the Enterprise domain you want to use with Instant Logs.
+
+1. Go to **Analytics** > **Instant Logs**.
```
Comment (marciocloudflare) [inline products/logs/src/content/instant-logs.md:16]: Where is this? I don't see this option in the dash

File: products/logs/src/content/instant-logs.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Click **Start streaming**.
+
+1. Narrow down on events by adding filters. To add filters, click on **Add filters**.
```
Comment (marciocloudflare) [inline products/logs/src/content/instant-logs.md]:
```suggestion
1. Click **Add filters** to narrow down the events shown.
```

Comment (marciocloudflare) [reply products/logs/src/content/instant-logs.md]: I think it's better to simplify the sentence. What do you think?

File: products/logs/src/content/instant-logs.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Once a filter is selected and the stream has started, only log lines that match the filter criteria will appear. Filters are not applied retroactively to logs already showing in the dash.
+
+## Datasets available
+
+For the moment, HTTP is the only dataset available. In the future, we will expand to other datasets.  
+
+## Exporting
+
+You can download the table of logs that appears in your dash using the **Export** button. The data will be downloaded in a JSON format.
```
Comment (marciocloudflare) [inline products/logs/src/content/instant-logs.md]:
```suggestion
You can download the table of logs that appears in your dash using the **Export** button. The data will be downloaded in JSON format.
```

File: products/logs/src/content/instant-logs.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+## Limits
+
+There are two limits set in place:
```
Comment (marciocloudflare) [inline products/logs/src/content/instant-logs.md]:
```suggestion
Instant Logs have two limits set in place:
```

File: products/logs/src/content/instant-logs.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+* Only one active Instant Logs session per zone.
+* Maximum session time is 60 minutes.
+* If either of these limits are reached, the logs stream will automatically stop.
```
Comment (marciocloudflare) [inline products/logs/src/content/instant-logs.md]:
```suggestion
* Only one active Instant Logs session per zone.
* Maximum session time is 60 minutes.

If either of these limits are reached, the logs stream will automatically stop.
```

Comment (marciocloudflare) [reply products/logs/src/content/instant-logs.md]: You had the explanation in the unordered list, which made it seem like it was a 3rd limit

File: products/logs/src/content/instant-logs.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+## Connect with us
+
+Share your feedback with us by joining the [Cloudflare Developers community on Discord](https://discord.gg/h35x2dkuq8)!
```
Comment (marciocloudflare) [inline products/logs/src/content/instant-logs.md]:
```suggestion
Share your feedback with us by joining the [Cloudflare Developers community on Discord](https://discord.gg/h35x2dkuq8).
```

Comment (marciocloudflare) [reply products/logs/src/content/instant-logs.md]: It might be a good idea to expand on this and tell customers what kind of feedback. the team wants.

**Review by marciocloudflare — CHANGES_REQUESTED** [review]

File: products/logs/src/content/instant-logs.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
+---
+order: 115
+pcx-content-type: concept
+---
+
+# Instant Logs
+
+Instant Logs allows Cloudflare Enterprise customers to access a live stream of the traffic for their domain on the Cloudflare dashboard. Seeing data in real time allows you to investigate an attack, troubleshoot, debug or test out changes made to your network. Instant Logs is lightweight, simple to use and does not require any additional setup.
```
Comment (angelampcosta) [inline products/logs/src/content/instant-logs.md:8]: In a team critique, we decided to go with singular verb form for product names, even for plural nouns, like Logs or Instant Logs: https://wiki.cfops.it/display/Prod/Product+Name+and+Pluralization

File: products/logs/src/content/instant-logs.md (hunk 3 above)
Comment (angelampcosta) [reply products/logs/src/content/instant-logs.md:16]: This is not in production, you need to choose the burritobot to be able to see the option.

File: products/logs/src/content/instant-logs.md (hunk 8 above)
Comment (angelampcosta) [reply products/logs/src/content/instant-logs.md]: @tanushreesharma-cf Is there any particular type of feedback that we would like to have?

File: products/logs/src/content/instant-logs.md (hunk 4 above)
Comment (angelampcosta) [reply products/logs/src/content/instant-logs.md]: I agree.

**Review by marciocloudflare — APPROVED** [review]

File: products/logs/src/content/instant-logs.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+1. Select the Enterprise domain you want to use with Instant Logs.
+
+1. Go to **Analytics** > **Instant Logs**.
+
+1. Click **Start streaming**.
+
+1. Click **Add filters** to narrow down the events shown.
+
+The filters you can add are **Firewall action matches**, **Country**, **Path**, **Status code**, **Client IP, Host**, **HTTP Method** and **Firewall Rule ID matches**. If you would like to see filtering on additional criteria, leave us feedback on the [form](https://docs.google.com/forms/d/e/1FAIpQLSduDhqhryhOYwUolDzXiLPNy5P07EY_abnYAFF_n7CRz4OzOQ/viewform) linked on the Instant Logs page.
```
Comment (tanushree-sharma) [inline products/logs/src/content/instant-logs.md]: Should the filters be capitalized?

File: products/logs/src/content/instant-logs.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+Once a filter is selected and the stream has started, only log lines that match the filter criteria will appear. Filters are not applied retroactively to logs already showing in the dash.
+
+## Datasets available
+
+For the moment, HTTP is the only dataset available. In the future, we will expand to other datasets.
```
Comment (tanushree-sharma) [inline products/logs/src/content/instant-logs.md]: This should be "For the moment, HTTP requests is the only dataset available."

**Review by tanushree-sharma — APPROVED** [review]

**Pull request merged**